webapp/src/DOMJudgeBundle/Resources/views/partials/scoreboard_table.html.twig

```diff
@@ -323,3 +337,49 @@
         </table>
     {% endif %}
 {% endif %}
+
+<style>
+    {% for color,i in backgroundColors %}
+    {% set cssColor = "#"~color %}
```

Comment:
Should we indent this one more time?

Reply:
Resolved in f7ccd52cd87e43d97000bd0aca80af5607a28160